src/ampPage.js

```diff
@@ -1,6 +1,11 @@
 import ampScripts from './ampScripts';
 import ampJson from './ampJson';
 
+function ampExperimentToken(token) {
+  if (!token) return '';
+  return `<meta name="amp-experiment-token" content="${token}">\n`;
```

Comment:
@ljharb if we are concerned about injection from values which should only be hardcoded by the programmer, then we need a PR to sanitize all of the options in this lib.